edenia/antelope-tools
# Pages insights score 644

### Apply some recommendations of the lighthouse

### What does this PR do?

- Resolve #644 

https://pagespeed.web.dev/report?url=https%3A%2F%2Feos.antelope.tools%2Fblock-producers&form_factor=desktop

Resolve the next issues
![imagen](https://user-images.githubusercontent.com/66583677/205396970-f1fed7a7-075a-4325-bc87-afd4b2bbee36.png)


- Using webp images in the network selector.
- Increase the size of network logos to display images with the correct aspect ratio.
- Check that bp logos are square and use https to avoid mixed content.
- Format code and remove unnecessary divs.

### Steps to test

1. Run the project locally.
1. Go to the block producers page. 
1. Run Lighthouse and verify that best practices and accessibility improved.
